projects/vdk-plugins/vdk-notebook/tests/jobs/ingest-job-sql-error/steps.ipynb

```diff
@@ -58,25 +41,19 @@
    "metadata": {
     "tags": [
      "vdk"
-    ]
-   },
-   "outputs": [],
-   "source": [
-    "response = requests.get(\"https://jsonplaceholder.typicode.com/todos/1\")"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "654db028-2d0e-4f95-a377-e1fac1255f08",
-   "metadata": {
-    "tags": [
-     "vdk"
-    ]
+    ],
```

Comment:
it was automatically added from my side, I will fix it now

Reply:
I will still have them, bause I get json format error when I remove them, they are generated because I fixed the notebooks from my PyCharm not jupyterlab